### DISCUSSION TITLE
Raspberry Pi Zero W

### DISCUSSION THREAD

**Discussion by jodfie** [jdpdev/birbcam#27] [Q&A]
Do you think this is too power hungry to be able to run on a Raspberry Pi Zero W?

**Comment by jdpdev** [comment]
I done a few super preliminary tests on a zero and it seems to run alright... it's noticeably slower but doesn't seem to effect actually taking pictures. I have some streamlining ideas that should help, but that's a future project

**Reply by jodfie** [reply]
Thanks. Does V2 Birbvision work on the Zero?

Couple more final questions...
Can you use a USB Webcam instead?

Can you give all in one place instructions to get 2.0birbcam birbvision and birbserver running?￼

I'll convert it to a markdown wiki page for you after if you want...

**Comment by jdpdev** [comment]
Sorry for the delay but I added documentation to birbcam and birbserver. Make sure you have the v2 branches for both repos.

* I _think_ the zero can run v2 but I haven't really tested it so ymmv
* Right now birbcam only supports the official raspi camera module. The v3 roadmap includes an extensive refactor to allow other camera sources, either in v3 or v4
